src/crc32.c

```diff
@@ -142,10 +149,13 @@ bool generic_crc32(target_s *const target, uint32_t *const result, const uint32_
 
 bool generic_crc32(target_s *const target, uint32_t *const result, const uint32_t base, const size_t len)
 {
-	uint8_t bytes[128U];
+	uint8_t bytes[1024U]; /* ADIv5 MEM-AP AutoInc range */
```

Comment:
Ideally we'd re-use a buffer like the GDB packet buffer here, but we'll go with this for now as "probably safe".

Reply:
As said, that idea of yours is harder in execution, I'll leave it up to you to try to implement it and reduce stack used. Use my meminfo branch to check clobbered depth (it colors the stack at reset).
It's just I tested swlink and it does not crash when doing this against a simple MCU target, but it may behave differently with multicore and multi-region targets like some MPUs.
And memory-map.xml generator may have to keep its buffer separate from gdb packet buffer to be able to hexify and print it (discussion idea).